pkgs/development/java-modules/jogl/default.nix

```diff
@@ -1,6 +1,79 @@
-{ lib, stdenv, fetchgit, ant, jdk8, git, xorg, udev, libGL, libGLU }:
+{ coreutils, lib, stdenv, fetchgit, ant, jdk8, jdk11, git, xorg, udev, libGL, libGLU, mesa, xmlstarlet }:
 
 {
+  jogl_2_4_0 =
+    let
+      version = "2.4.0";
+
+      gluegen-src = fetchgit {
```

Comment:
Looks like this derivation has more than one `src`, therefore, I would suggest to use `srcs`.

Reply:
Thank you, TIL.